pnpm-lock.yaml

```diff
@@ -8018,6 +8018,16 @@ packages:
       zod: ^3.18.0
     dependencies:
       zod: 3.22.4
+    dev: true
+
+  /zod-validation-error@3.0.2(zod@3.22.4):
+    resolution: {integrity: sha512-21xGaDmnU7lJZ4J63n5GXWqi+rTzGy3gDHbuZ1jP6xrK/DEQGyOqs/xW7eH96tIfCOYm+ecCuT0bfajBRKEVUw==}
+    engines: {node: '>=18.0.0'}
+    peerDependencies:
+      zod: ^3.18.0
+    dependencies:
+      zod: 3.22.4
+    dev: false
```

Comment:
Noting that Knip has `3.0.0` exactly. Once they bump this duplication will go away.